### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by janb87**

### CODE REVIEW

File: src/commands/build.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ module.exports.handler = function() {
             configContents[CONSTANTS.CONFIG_CHAINCODES_KEY]
         ).then(() => {
             console.log('Done building!');
+        }).catch((error) => {
+            console.error('Build failed with following error:', error)
```
Comment (daanporon) [inline src/commands/build.js]: isn't this error catched below?

File: src/templates/createChaincode/chaincode/package.json [hunk 2]
```diff
@@ -12,10 +12,6 @@
     "engine-strict": true,
     "engineStrict": true,
     "license": "MIT",
-    "dependencies": {
-        "@kunstmaan/hyperledger-fabric-node-chaincode-utils": "^0.1.0",
```
Comment (daanporon) [inline src/templates/createChaincode/chaincode/package.json]: i added this here as well and in the common for your IDE to know about these things and you can have code completion then ...

File: src/commands/build.js (hunk 1 above)
Comment (janb87) [reply src/commands/build.js]: This catch was not needed indeed. The unhandled promise rejection was at another place

File: src/templates/createChaincode/chaincode/package.json (hunk 2 above)
Comment (janb87) [reply src/templates/createChaincode/chaincode/package.json]: Restored it

**Pull request merged**